website/src/components/Chat/ChatConfigSaver.tsx

```diff
@@ -4,13 +4,15 @@ import { ChatConfigFormData } from "src/types/Chat";
 import { setConfigCache } from "src/utils/chat";
 
 export const ChatConfigSaver = () => {
-  const { watch, formState } = useFormContext<ChatConfigFormData>();
+  const { watch, formState, reset } = useFormContext<ChatConfigFormData>();
   const config = watch();
   useEffect(() => {
     if (formState.isDirty) {
       setConfigCache(config);
+      // unset isDirty
+      reset(config);
```

Comment:
I think you a have bad merge here, I removed this when I fix a bug related to storing the config in the local storage.